tests/sdks/python/infrastructure/test_secret_provider_factory.py

```
@@ -94,7 +94,7 @@ def Should_RequireVaultUrl_When_AzureProviderSelected(
 
     def Should_RaiseValueError_When_ConfigIsNone(self) -> None:
         # Act
-        action = lambda: SecretProviderFactory.create(None)
+        action = lambda: SecretProviderFactory.create(None)  # type: ignore[arg-type]
```

Ruff (0.15.10)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `action` as a `def`

(E731)